app/controllers/api/v0/tasks_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Api
+  module V0
+    class TasksController < ApplicationController
+      include Api::Mixins::IndexMixin
+      include Api::Mixins::ShowMixin
+
+      def list_params
+        params.permit(:tenant_id)
```

Comment:
permit or require?

Reply:
I think this needs to be figured out generically.  It's supposed to be in a header, right?